### ISSUE TITLE
Procedures created and accessed in different schemas

### ISSUE THREAD

**Issue by jerolba** [torodb/torodb#20]
In a clean database, when ToroDb starts, creates procedures in torodb schema, 

![image](https://cloud.githubusercontent.com/assets/709055/7987715/648d72d4-0adf-11e5-96f7-cfd5afb62d31.png)

but when intances a FirstFreeDocId class the schema used is PUBLIC

```java
	public FirstFreeDocId() {
		super("first_free_doc_id", com.torodb.torod.db.postgresql.meta.PublicSchema.PUBLIC, org.jooq.impl.SQLDataType.INTEGER);

		setReturnParameter(RETURN_VALUE);
		addInParameter(COL_SCHEMA);
	}
```

The log error is:

```
Caused by: java.lang.RuntimeException: org.jooq.exception.DataAccessException: SQL [{ ? = call "public"."reserve_doc_ids"(?, ?) }]; ERROR: no existe la función public.reserve_doc_ids(character varying, integer)
  Hint: Ninguna función coincide en el nombre y tipos de argumentos. Puede ser necesario agregar conversión explícita de tipos.
```

**Comment by gortiz** [comment]
Are you sure you didn't modify your torodb sources? We want to create the procedures on torodb schema, but right now they are created on public schema. In fact, all SQL structures (procedures, types, casts) are created executing SQL scripts stored on [torod/db-wrappers/postgresql/src/main/resources/sql](https://github.com/torodb/torodb/tree/master/torod/db-wrappers/postgresql/src/main/resources/sql). As you can see there, in both devel and master branch, no schema is used.

**Comment by jerolba** [comment]
No, I didn't change any source. I just cloned the master branch, executed mvn package and started toroDb with the command:

java -jar torodb-0.22.1-jar-with-dependencies.jar -d mongodb -p 5433 -u torodb --ask-for-password

**Comment by gortiz** [comment]
This issue has been fixed on 7182f1e
